### PR TITLE
Add employment history to pensions form

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3021

screen shot 2017-05-31 at 2 34 48 pm
screen shot 2017-05-31 at 2 34 51 pm
screen shot 2017-05-31 at 2 34 33 pm

### CODE REVIEW

**Review by cehsu — APPROVED** [review]
LGTM.

File: src/js/pensions/config/form.js [hunk 1]
```diff
@@ -184,6 +189,52 @@ const formConfig = {
               }
             }
           }
+        },
+        employmentHistory: {
+          title: 'Employment history',
+          path: 'employment/history',
+          depends: isUnder65,
+          uiSchema: {
+            'ui:description': 'Please tell us about all of your employment, including self-employment, from one year before you became disabled to the present.',
```
Comment (cvalarida) [inline src/js/pensions/config/form.js]: Should "from one year before you became disabled" be emboldened?

File: src/js/pensions/components/EmploymentField.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import moment from 'moment';
+
+export default function FullNameField({ formData }) {
```
Comment (cvalarida) [inline src/js/pensions/components/EmploymentField.jsx]: Component name doesn't match file name. I think you meant to change this to `EmploymentField` after copying. 😉

File: src/js/pensions/config/form.js [hunk 3]
```diff
@@ -184,6 +189,52 @@ const formConfig = {
...
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              jobs: {
+                type: 'array',
+                minItems: 1,
+                items: {
+                  type: 'object',
+                  required: ['address', 'employer', 'jobTitle', 'dateRange', 'daysMissed', 'annualEarnings'],
```
Comment (cvalarida) [inline src/js/pensions/config/form.js]: @goldenmeanie I assume all this is right, but are all the fields from the original ticket required?

File: test/pensions/config/employmentHistory.unit.spec.jsx [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+      target: {
+        value: '2003'
+      }
+    });
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_jobs_0_daysMissed'), {
+      target: {
+        value: '3'
+      }
+    });
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_jobs_0_annualEarnings'), {
```
Comment (cvalarida) [inline test/pensions/config/employmentHistory.unit.spec.jsx]: We may consider making a wrapper for this. It's quite verbose.

**Review by cvalarida — COMMENTED** [review]
Mmm...In the screenshot, the country is USA but it says "Postal code", not "Zipcode". Is that right?

File: src/js/pensions/config/form.js (hunk 3 above)
Comment (jbalboni) [inline src/js/pensions/config/form.js]: Yeah, I'm not sure why I did that exactly. Should confirm that.

File: test/pensions/config/employmentHistory.unit.spec.jsx (hunk 4 above)
Comment (jbalboni) [inline test/pensions/config/employmentHistory.unit.spec.jsx]: Yeah. I wonder how much effort we want to put into that, vs. just deciding to switch over to Enzyme.

**Comment by jbalboni** [comment]
@cvalarida We always say postal code in the common address definition.

File: src/js/pensions/config/form.js (hunk 1 above)
Comment (jbalboni) [inline src/js/pensions/config/form.js]: Yep

File: src/js/pensions/config/form.js (hunk 3 above)
Comment (jbalboni) [inline src/js/pensions/config/form.js]: Confirmed that they should be required and updated the issue.

File: test/pensions/config/employmentHistory.unit.spec.jsx (hunk 4 above)
Comment (cvalarida) [inline test/pensions/config/employmentHistory.unit.spec.jsx]: Ohh, yeah. I forgot about Enzyme.

**Pull request merged**